Makefile

```diff
@@ -20,7 +30,7 @@ ENVTEST_K8S_VERSION = 1.31.0
 ENVTEST_VERSION ?= release-0.19
```

Comment:
Why release-0.19 ?

Reply:
This version of `setup-envtest` is compatible with `ENVTEST_K8S_VERSION = 1.31.0`